docs/tutorials/code/chisel/task.yaml

```diff
@@ -23,11 +23,11 @@ execute: |
   # [docs:skopeo-copy-end]
 
   # [docs:docker-run]
-  docker run --rm  chisel-openssl
+  docker run --rm chisel-openssl exec openssl
   # [docs:docker-run-end]
 
   # [docs:docker-run-with-args]
-  docker run --rm chisel-openssl s_client -connect ubuntu.com:443 -brief
+  docker run --rm chisel-openssl exec --env=SSL_CERT_FILE=/etc/ssl/certs/ca-certificates.crt openssl s_client -connect ubuntu.com:443 -brief
```

Comment:
Hmm. Since this is a tutorial, I'm wondering about the best way to do this. Instead of `exec --env...`, wouldn't it be preferable to define an `openssl` pebble service with the `SSL_CERT_FILE` env var?
Is the issue passing the remaining parameters to the `openssl` service?

Reply:
yes you can't do this today. It is a Pebble limitation. whether it is by design or not, we'll have to take it internally for further discussions